.github/workflows/test.yml

```diff
@@ -1,5 +1,5 @@
 name: build
-on: [push, pull_request_target]
+on: [push, pull_request]
```

Comment:
This fixes a [regression from #1722](https://github.com/cesanta/mongoose/pull/1722#discussion_r982966134).